python/mspasspy/algorithms/snr.py

```diff
@@ -13,7 +13,7 @@
     EstimateBandwidth,
     BandwidthStatistics,
 )
-from mspasspy.ccore.algorithms.basic import TimeWindow, Butterworth, ExtractComponent
+from mspasspy.ccore.algorithms.basic import TimeWindow, Butterworth, ExtractComponentSeismogram
```

Comment:
Should have said this in the previous box.  Unless we can overload an `_ExtractComponent` function too avoid two names, I think we should merge this fix.  However, there is more to fix in the issues page this references.  There is an issue with the aspect ration in imageplot that will take some more digging.  That is less critical as the tutorial runs.  The problem happens for plots of small numbers of items that show nothing useful because the plot is a thin, black strip.

Reply:
While overloading is not supported, we can still implement a `ExtractComponent` function in Python that calls the different underlying variant when different object type is given. This will be much less confusing than having the two functions.